tests/foreman/ui/test_registration.py

```diff
@@ -101,20 +110,16 @@ def test_positive_org_loc_change_for_registration(
             assert pair in cmd
 
 
-def test_negative_global_registration_without_ak(
-    module_target_sat,
-    module_org,
-    module_location,
-):
+def test_negative_global_registration_without_ak(target_sat, function_org, function_location):
     """Attempt to register a host without ActivationKey
 
     :id: 34122bf3-ae23-47ca-ba3d-da0653d8fd36
 
     :expectedresults: Generate command is disabled without ActivationKey
     """
-    with module_target_sat.ui_session() as session:
-        session.organization.select(org_name=module_org.name)
-        session.location.select(loc_name=module_location.name)
+    with target_sat.ui_session() as session:
```

Comment:
We can continue to use module_target_sat, we'll just need func org and loc for this test
```suggestion
    with module_target_sat.ui_session() as session:
```

Reply:
Updated.